cpp/test/test_utils.cpp

```diff
@@ -0,0 +1,70 @@
+#include <random>
+#include <vector>
+
+#include "array_utils.h"
+
+NDArray<float, 2> randomQuantizedVectorsNDArray(int numVectors,
```

Comment:
I think we can remove these functions and instead just use the `randomQuantizedVectors` and `randomVectors` methods